backend/controller/cronjobs/sql/models.go

```diff
@@ -478,6 +478,14 @@ type ModuleConfiguration struct {
 	Value     []byte
 }
 
+type ModuleSecret struct {
+	ID        int64
+	CreatedAt time.Time
+	Module    optional.Option[string]
+	Name      string
+	Url       string
+}
```

Comment:
I'm guessing this is unrelated, but needed to be checked in?

Reply:
It is related to code generated in the CI causing the repo to be dirty. I'm under the impression these sqlc files needed to be checked in?

Reply:
Your impression is correct :) Looks like whoever added ModuleSecret to the db schema just didn't know they needed to run `just build-sqlc`.